packages/multimedia/intel-vaapi-driver/package.mk

```
@@ -11,15 +11,17 @@ PKG_SITE="https://01.org/linuxmedia"
 PKG_URL="https://github.com/intel/intel-vaapi-driver/archive/$PKG_VERSION.tar.gz"
 PKG_DEPENDS_TARGET="toolchain libva libdrm"
 PKG_LONGDESC="intel-vaapi-driver: VA-API user mode driver for Intel GEN Graphics family"
-PKG_TOOLCHAIN="autotools"
+PKG_TOOLCHAIN="meson"
 
 if [ "$DISPLAYSERVER" = "x11" ]; then
-  DISPLAYSERVER_LIBVA="--enable-x11 --disable-wayland"
+  DISPLAYSERVER_LIBVA="-Dwith_x11=yes -Dwith_wayland=no"
 elif [ "$DISPLAYSERVER" = "weston" ]; then
-  DISPLAYSERVER_LIBVA="--disable-x11 --enable-wayland"
+  DISPLAYSERVER_LIBVA="-Dwith_x11=yes -Dwith_wayland=yes"
```

`-Dwith_x11=yes` looks wrong for weston (untested)

```suggestion
  DISPLAYSERVER_LIBVA="-Dwith_x11=no -Dwith_wayland=yes"
```

---

Thanks!